src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/media-edit/main.js

```diff
@@ -421,14 +508,20 @@ define(function () {
          * Maps the overlay input back on multiple models
          * @returns {Boolean} returns false if form is invalid, true if valid
          */
-        changeMultipleModel: function () {
+        changeMultipleModel: function() {
             if (this.sandbox.form.validate(constants.multipleEditFormSelector)) {
-                var data = this.sandbox.form.getData(constants.multipleEditFormSelector);
+                var data = this.sandbox.form.getData(constants.multipleEditFormSelector),
```

Comment:
Is this the only difference to the function `changeSingleModel`? The this could be solved in a more elegant way.